<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomItemWidgetMeshRegions1Class</class>
 <widget class="QWidget" name="CustomItemWidgetMeshRegions1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>67</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CustomItemWidgetMeshRegions1</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">        QComboBox {
            border: none;
            background-color: transparent;
            padding: 5px;
        }
        QComboBox:hover {
            background-color: rgb(242, 242, 242);
            border: 1.5px solid lightgray;
        }
        QComboBox::drop-down {
            border: none;
        }
        QComboBox::down-arrow {
            image: url(../res/Options.png); 
            width: 12px;
            height: 12px;
        }
        QComboBox::down-arrow:on {
            top: 1px;
            left: 1px;
        }
        QComboBox::item {
            padding-left: 0px; /* 隐藏左边的图标 */
        }
        QComboBox QAbstractItemView {
            border: 1px solid lightgray;
            selection-background-color: rgb(242, 242, 242);
            outline: none; /* 移除虚线框 */
        }
        QComboBox QAbstractItemView::item {
            padding: 5px;
            min-height: 25px;
            outline: none; /* 移除虚线框 */
        }
        QComboBox QAbstractItemView::item:hover {
            background-color: rgb(242, 242, 242);
            color: black; /* 确保鼠标悬浮时字体颜色为黑色 */
        }
        QComboBox QAbstractItemView::item:selected {
            background-color: rgb(242, 242, 242);
            color: black; /* 确保选中时字体颜色为黑色 */
        }
        QComboBox::item:selected {
            background-color: rgb(242, 242, 242);
        }
        QComboBox::item:icon {
            padding-left: 0px; /* 隐藏左边的图标 */
        }</string>
       </property>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
